website/src/pages/customers.tsx

```diff
@@ -356,57 +308,22 @@ export default ((props: any) => (
-                </div>
-                <div className="row justify-content-center pt-6">
-                    <div className="col-lg-10 text-center">
-                        <img src="/external-logos/sofi-logo.svg" width="100px" />
-                        <blockquote className="blockquote case-studies__quote case-studies__quote--in-content">
-                            <p>
-                                Sourcegraph is an ingrained part of my daily process at SoFi. With Sourcegraph, our
-                                engineers can understand all of the repercussions of committing a change to a service
-                                that is exposed to other services.
-                            </p>
-                            <footer className="blockquote-footer">Ursula Robertson, Engineering Manager, SoFi</footer>
-                        </blockquote>
-                        <Link to="/case-studies/sofi-moves-fast-on-hundreds-of-microservices/">
-                            SoFi adopts Sourcegraph Universal Code Search to manage hundreds of microservices
-                            <ArrowRightBoxIcon className="icon-inline ml-1" />
-                        </Link>
-                    </div>
-                </div>
-                <hr className="my-6" />
-            </ContentSection>
-            <ContentSection className="my-6" id="campaigns">
+            <ContentSection className="my-6" id="code-reviews">
                 <div className="row">
                     <div className="col-lg-5 mt-5">
-                        <h2 className="display-3 font-weight-bold">Automated code changes > manual</h2>
+                        <h2 className="display-3 font-weight-bold">Catch more bugs with better code reviews</h2>
                         <p>
-                            Move quickly and safely when removing legacy code, fixing critical security issues, and
-                            paying down tech debt. Identify which call sites might be affected by changes, and run
-                            campaigns to compute diffs and create branches and pull requests across multiple
-                            repositories. Large-scale code changes and refactors can be painless, more accurate, and
-                            faster, when done automatically with Sourcegraph
+                            Mistakes that slip past code review are 10x harder to fix. Make code reviews fast, thorough,
+                            and less painful — no more tl;dr – and catch bugs before build. Sourcegraph’s code intelligence
+                            paired with live data pulled from external services means code reviews are done right.
                         </p>
-                    </div>
+                    </div>{' '}
                     <div className="col-lg-7 mt-5 pl-lg-6">
+                        <h5>Watch below to see how Sid Sijbrandij (GitLab CEO) and Quinn Slack (Sourcegraph CEO) do code reviews.</h5>
```

Comment:
```suggestion
                        <h5>Watch how Sid Sijbrandij (GitLab CEO) and Quinn Slack (Sourcegraph CEO) do code reviews.</h5>
```

Reply:
added the extra text because the second line gets a bit short when the screen is between 770ish-1000px and above 1200 px... but the suggested wording does read better.

Reply:
I can do a responsive review, once all the copy is approved.